includes/definitions/adva_fsp150.yaml

```diff
@@ -0,0 +1,13 @@
+os: adva_fsp150
+text: ADVA_FSP150CC
```

Comment:
These are display texts so you should do them as human readable like Adva FSP150CC